packages/system-tests/src/spectron/application.ts

```diff
@@ -61,7 +61,8 @@ export class SpectronApplication {
       path: electronPath,
       args: args,
       chromeDriverArgs: chromeDriverArgs,
-      startTimeout: 10000
+      startTimeout: 10000,
+      requireName: 'nodeRequire'
```

Comment:
Is this the change from the VS Code team?

Reply:
Yes they added this to their spectron framework.